app/views/recipies/show.html.erb

```diff
@@ -0,0 +1,52 @@
+<section>
+<h2 class="recipe-head">
+  <%= @recipy.name%>
+</h2>
+<br />
+<div class="recipe-info">
+<p>
+  preparation time: <%= @recipy.preparation_time%>
+</p>
+<p>
+   cooking time: <%= @recipy.cooking_time%>
```

Comment:
- [ ] Your recipe details page should be like on the wireframe provided in the project requirement. Kindly, make sure that your details page looks similar as on the screenshot below:
![image](https://user-images.githubusercontent.com/71874425/175667794-f0386341-53c9-47bf-9b98-e8ad6fb672eb.png)

Reply:
Done